src/components/dropdown/DropDown.js

```diff
@@ -1,54 +1,77 @@
+        const {onSelect} = this.props
 
+        // todo:
+        //  объявить переменную value, записать в нее  event.target.getAttribute('data-value');
+        //  вывести console.log(value) и убедиться что в консоли показывается текст выбранной опции
+        const value = event.target.getAttribute('data-value')
+        console.log(value)
 
-    // todo:
-    //  объявить переменную value, записать в нее  event.target.getAttribute('data-value');
-    //  вывести console.log(value) и убедиться что в консоли показывается текст выбранной опции
+        // todo: вызвать функцию из onSelect с аргументом value
+        onSelect(value)
 
+        // todo: закрыть дропдаун, вызвав this.setState({ isOpen: false })
+        this.setState({isOpen: false})
+    };
 
-    // todo: вызвать функцию из onSelect с аргументом value
+    render() {
+        // todo:
+        //  достать из this.props массив опций options
+        //  он должен быть по умолчанию пуст, т.е. = [] (дефолтный параметр при деструктуризации) и
+        //  достать также в пропсах selectedOption (выбранная в данный момент опция)
 
+        const {options = [], selectedOption} = this.props
+        // todo:
+        //  достать isOpen из this.state
+        const {isOpen} = this.state
+        return (
+            <div className="may-drop-down dropdown" onClick={this.toggle}>
+                {/* todo: показать в строке 45 выбранную опцию selectedOption  и передать в onClick this.toggle
 
-    // todo: закрыть дропдаун, вызвав this.setState({ isOpen: false })
-  };
-
-  render() {
-    // todo:
-    //  достать из this.props массив опций options
-    //  он должен быть по умолчанию пуст, т.е. = [] (дефолтный параметр при деструктуризации) и
-    //  достать также в пропсах selectedOption (выбранная в данный момент опция)
-
-    // todo:
-    //  достать isOpen из this.state
-
-    return (
-      <div className="may-drop-down dropdown">
-        {/* todo: показать в строке 45 выбранную опцию selectedOption  и передать в onClick this.toggle
              (нажатие на этот тег должно открывать/закрывать дропдаун)
         */}
-        <div className="dropdown-toggle">{/* тут будет выбранная опция*/}</div>
-
-        {
-          // todo:
-          //  если значение isOpen в this.state = true показываем этот блок ниже (использовать &&)
-          <div className="may-drop-down-options-wrapper dropdown-menu show">
-            {/*
+                <div className="dropdown-toggle">{selectedOption}</div>
+                    // todo:
+                    //  если значение isOpen в this.state = true показываем этот блок ниже (использовать &&)
+                   {isOpen && (
+                    <div className="may-drop-down-options-wrapper dropdown-menu show">
+                        {options.map((option) => {
+                            let itemClass = "may-drop-down-options-wrapper-option dropdown-item"
+                            if (selectedOption === option) {
```

Comment:
This is good, also it can be done using a string literal
no need to fix this, just keep in mind for the next time that it's also possible